src/Utc.roc

```diff
@@ -53,3 +54,10 @@ deltaAsMillis = \@Utc first, @Utc second ->
 deltaAsNanos : Utc, Utc -> U128
 deltaAsNanos = \@Utc first, @Utc second ->
     Num.absDiff first second
+
+## Sleep for a given number of milliseconds
+sleepMillis : U64 -> Task {} *
+sleepMillis = \millis ->
```

Comment:
Thinking ahead, I think this would be better as `Sleep.millis` than `Utc.sleepMillis` - because `Utc` should probably be a shared module with no effects in it, and when we have effects shared across platforms, I think it would be best to have the "sleep" effect in its own module so it doesn't end up requiring that a particular support more effects than necessary to offer it!